libraries/botbuilder-dialogs/src/skillDialog.ts

```diff
@@ -59,7 +58,7 @@ export class SkillDialog extends Dialog {
         this.dialogOptions = dialogOptions;
     }
 
-    public async beginDialog(dc: DialogContext, options?: any): Promise<DialogTurnResult> {
+    public async beginDialog(dc: DialogContext, options: BeginSkillDialogOptions): Promise<DialogTurnResult> {
```

Comment:
This is a slight breaking change, but as we validated that `options` matched our requirements with the private helper `validateBeginDialogArgs` it should be fine.

@gabog, @johnataylor, if you have concern about this change in the surface API now would be a good time to chime in...